src/main/java/ch/hftm/controllers/AlbumController.java

```diff
@@ -95,7 +122,60 @@ private void createLabel(GridPane pane, int column, String labelText) {
         }
     }
 
+    @FXML
+    public void deleteImage() {
+        Alert alert = new Alert(AlertType.CONFIRMATION);
+        alert.setTitle("Delete selected images");
+        alert.setContentText("If you want to delete the selected images press OK.");
+        ButtonType okButton = new ButtonType("OK");
+        ButtonType cancelButton = new ButtonType("Cancel");
+        alert.getButtonTypes().setAll(okButton, cancelButton);
+        Optional<ButtonType> choice = alert.showAndWait();
+
+        if (choice.isPresent() && choice.get() == okButton) {
+            ImageService imageService = new ImageService();
```

Comment:
We are using the ImageService multiple types in this class, maybe we could create a class level property which holds the imageService and instantiate it in the initialize or the constructor, so it is available in all methods without having to create it everytime

Reply:
A class level ImageService works for me.